### PR TITLE
style(frontend): NetworksSwitcher is moved from the header to the tokens list

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

As recent requirements, we move the component `NetworkSwitcher` from the header to the tokens list.

# Changes

- Moved component `NetworkSwitcher` into `Tokens`.
- Created new UI component `ButtonSwitcher` with specific style and used in `NetworkSwitcher`.
- Adapted popover to appear left-to-right

# Results

Screenshot 2024-07-17 at 13 25 04
Screenshot 2024-07-17 at 13 25 11
Screenshot 2024-07-17 at 13 25 25
Screenshot 2024-07-17 at 13 25 52

### CODE REVIEW

File: src/frontend/src/lib/components/networks/NetworksSwitcher.svelte [hunk 1]
```diff
@@ -26,18 +25,15 @@
...
 	{disabled}
-	imgSrc={$selectedNetwork?.icon ?? chainFusion}
-	imgAlt={replacePlaceholders($i18n.core.alt.logo, {
-		$name: $selectedNetwork?.name ?? $i18n.networks.chain_fusion
-	})}>{$selectedNetwork?.name ?? $i18n.networks.chain_fusion} <IconChevronDown /></ButtonHero
+	>{$selectedNetwork?.name ?? $i18n.networks.chain_fusion} <IconChevronDown /></ButtonSwitcher
 >
 
-<Popover bind:visible anchor={button} direction="rtl">
+<Popover bind:visible anchor={button} direction="ltr">
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/networks/NetworksSwitcher.svelte:36]: `ltr` is the default maybe?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

Note: there is maybe some logic that redirects the user to the tokens page if the network is changed on another page, like on transactions or settings page. Should we keep that logic or clean it up in another PR?

File: src/frontend/src/lib/components/networks/NetworksSwitcher.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/networks/NetworksSwitcher.svelte:36]: yes, you are right, just checked. I will remove it

**Comment by AntonioVentilii-DFINITY** [comment]
Well, since it is a "insurance" logic (right?) I would keep it: if something should happen, for whatever reason, that the network is changed in another page, it is always better to fallback to the main list, no?

**Comment by peterpeterparker** [comment]
Makes sense.

**Pull request merged**